languagetool-language-modules/en/src/main/resources/org/languagetool/rules/en/grammar.xml

```diff
@@ -83369,6 +83369,48 @@ The accident victim died from her injuries.
                 <example>He <marker>is baking</marker> a cake.</example>
             </rule>
         </rulegroup> <!-- end passive voice rule -->
+		<rule id="OCCASION_TRANSITIVE_VERB_ARCHAIC" name="transitive verb 'occasion' is archaic" default="temp_off">
+			<antipattern>
+				<token postag="N.*" postag_regexp="yes" inflected="yes">occasion</token>
+			</antipattern>
+			<!-- Avoids false positives caused by the VBZNNS_NNS disambiguator incorrectly returning
+			only VBZ for 'occasions'. See the set of examples marked (*) below. -->
+			<antipattern>
+				<token>on</token>
+				<token postag="JJ|CD|RB" postag_regexp="yes" min="0" max="3" />
+				<token inflected="yes">occasion</token>
+			</antipattern>
+			<pattern>
+				<marker><token postag="VB.*" postag_regexp="yes" inflected="yes">occasion</token></marker>
+			</pattern>
+			<message>The transitive verb "<match no="1" />" is an archaic word choice. Consider a more common synonym, such as <!--
+				--><suggestion><match no="1" postag="VB.*">cause</match></suggestion>, <!--
+				--><suggestion><match no="1" postag="VB.*">induce</match></suggestion>, or <!--
+				--><suggestion><match no="1" postag="VB.*">bring</match> about</suggestion>
+			</message>
+			<url>https://tinyurl.com/yyxj28z8</url>
+			<example correction="caused|induced|brought about">The concert <marker>occasioned</marker> a riot.</example>
+			<example correction="caused|induced|brought about">It was <marker>occasioned</marker> by it.</example>
+			<!-- Not sure yet how to code a suggestion like 'bring [direct object] about'
+			(suggestion should give 'bring it about', not just 'bring about').
+			Would I need to extend the pattern <marker> around the entire direct object? -->
```

Comment:
Yes, you would indeed have to put both the predicate and the direct object inside the pattern. My impression is that English avoids excessively long "brackets" and only requires _about_ to be put to the right of the object if it is one (or two?) words long. Is that so?

_It was used to occasion it._ --> It was used to bring it about.
_It was used to occasion those changes._ --> It was used to bring those changes about / bring about those changes.
_It was used to occasion those long-needed changes in our society._ --> It was used to bring about those long-needed changes in our society (not: It was used to **bring** those long-needed changes in our society **about**)